torchgeo/datasets/oscd.py

```diff
@@ -391,14 +397,21 @@ def __init__(
             self.band_stds = self.band_stds[:, None, None]
 
         self.norm = Normalize(self.band_means, self.band_stds)
+        self.rcrop = K.AugmentationSequential(
+            K.RandomCrop(crop_size[1:]), data_keys=["input", "mask"], same_on_batch=True
+        )
 
     def preprocess(self, sample: Dict[str, Any]) -> Dict[str, Any]:
         """Transform a single sample from the Dataset."""
         sample["image"] = sample["image"].float()
+        sample["mask"] = sample["mask"].float()
         sample["image"] = self.norm(sample["image"])
         sample["image"] = torch.clamp(  # type: ignore[attr-defined]
```

Comment:
This clamp line should be removed. You are standardizing per channel, which means the data will have 0 mean and 1 stdev -- if you clamp then you are getting rid of > 1/2 of your data.